lib/persisted-model.js

```diff
@@ -1930,29 +1928,34 @@ module.exports = function(registry) {
   };
 
   /**
-   * Get the filter for searching related changes. Override this function, if need to use properties from the model
-   * filter as part the change search filter
+   * Get the filter for searching related changes.
    *
-   * example override for this function(in a mixin) when a custom property is used in change model, using model instance
-   * property value in the change filter.
+   * Models should override this function to copy properties
+   * from the model instance filter into the change search filter.
    *
+   * ```js
    * module.exports = (TargetModel, config) => {
-   *   var originalFilterFactory = TargetModel.createChangeFilter;
-   *
-   *   TargetModel.createChangeFilter = function (since, modelFilter) {
-   *     var customProperty = TargetModel.settings.changeCustomProperty;
-   *
-   *     var filter = originalFilterFactory.call(this, since, modelFilter);
-   *     if (modelFilter && modelFilter.where && modelFilter.where[customProperty]) {
-   *       filter.where[customProperty] = modelFilter.where[customProperty];
+   *   TargetModel.createChangeFilter = function(since, modelFilter) {
```

Comment:
Simplified the example - use hard-coded property name "tenantId", get the base `createChangeFilter` from the parent model.